src/lib/components/explorer/export/ExportStepper.svelte

```diff
@@ -222,63 +291,197 @@
-          {#if canDownload}
-            {#await statusPromise}
-              <div class="flex justify-center items-center">
-                <ProgressRadial width="w-4" />
-                <div>Prepareing your dataset...</div>
-              </div>
-            {:then status}
-              <div class="flex flex-col items-center justify-center">
+      <div class="flex justify-center">
+        {#if canDownload}
+          {#await statusPromise}
+            <div class="flex justify-center items-center">
+              <ProgressRadial width="w-4" />
+              <div>Preparing your dataset...</div>
+            </div>
+          {:then}
+            {#if query.query.expectedResultType === 'DATAFRAME'}
+              <section class="flex flex-col gap-8">
+                <p class="mt-4">
+                  To export data and start your analysis, copy and paste the following code in an
+                  analysis workspace, such as BioData Catalyst Powered by Seven Bridges or BioData
+                  Catalyst Powered by Terra, to connect to PIC-SURE and save the data frame or
+                  download the file. Note that you will need your personal access token to complete
+                  the connection to PIC-SURE with code.
+                </p>
+                <TabGroup class="card p-4">
+                  {#if query.query.expectedResultType === 'DATAFRAME'}
+                    <Tab bind:group={tabSet} name="python" value={tabIndex++}>Python</Tab>
+                    <Tab bind:group={tabSet} name="r" value={tabIndex++}>R</Tab>
+                  {/if}
+                  <Tab bind:group={tabSet} name="download" value={tabIndex++}>Download</Tab>
+                  <svelte:fragment slot="panel">
+                    {#if tabSet === 0}
+                      <CodeBlock
+                        language="python"
+                        lineNumbers={true}
+                        buttonCopied="Copied!"
+                        code={`# Requires python 3.7 or later
+import sys
+import pandas as pd
+import matplotlib.pyplot as plt
+# BDC Powered by Terra users uncomment the following line to specify package install location
+# sys.path.insert(0, r"/home/jupyter/.local/lib/python3.7/site-packages")
+!{sys.executable} -m pip install --upgrade --force-reinstall git+https://github.com/hms-dbmi/pic-sure-python-client.git
+!{sys.executable} -m pip install --upgrade --force-reinstall git+https://github.com/hms-dbmi/pic-sure-python-adapter-hpds.git
+!{sys.executable} -m pip install --upgrade --force-reinstall git+https://github.com/hms-dbmi/pic-sure-biodatacatalyst-python-adapter-hpds.git
+import PicSureHpdsLib
+import PicSureClient
+
+PICSURE_network_URL = "https://picsure.biodatacatalyst.nhlbi.nih.gov/picsure"
+
+token_file = "token.txt"
+with open(token_file, "r") as f:
+    my_token = f.read()
+
+connection = PicSureClient.Client.connect(url = PICSURE_network_URL, token = my_token)
+
+queryID = THIS SHOULD DISPLAY THE GENERATED DATASET ID
```

Comment:
I think you were supposed to fill this in with the dataset

Reply:
Oh, yeah I re-copied all of this because there were some changes and forgot to update again